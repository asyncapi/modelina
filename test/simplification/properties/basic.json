{
  "properties": { 
    "testProp1": { 
      "type": "string"
    },
    "testProp2": { 
      "type": "string"
    } 
  }
}