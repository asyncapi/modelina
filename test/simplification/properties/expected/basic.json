{
  "testProp1": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  },
  "testProp2": {
    "type": "string",
    "originalSchema": {
      "type": "string"
    }
  }
}